startup_runners/Makefile

```diff
@@ -19,14 +19,14 @@ startup_runner.class: startup_runner.java
 
 
 outer_startup_runner_c: outer_startup_runner.c
-	${CC} ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
-		${CPPFLAGS} outer_startup_runner.c \
+	${CC} --std=c99 ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
```

Comment:
That's weird.

Anyway, if those are the only files affected, let's keep the gnu99 to just those. These files are fine as "normal" c99 IMHO.